plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/MigrateProcedure.java

```diff
@@ -48,6 +48,7 @@
 import io.trino.spi.type.ArrayType;
 import io.trino.spi.type.MapType;
 import io.trino.spi.type.RowType;
+import io.trino.spi.type.TimestampType;
```

Comment:
i'd suggest renaming `Support timestamp(3) type in Iceberg migrate procedure` commit to `Support timestamp type in Iceberg migrate procedure`